src/localization/locales/English.json

```diff
@@ -128,6 +128,8 @@
   "no.addresses.yet": "No addresses yet",
   "no.groups.yet": "No groups yet",
 
+  "my.assets":"My assets",
```

Comment:
Can you add anything you add in English.json to Turkish.json without any translations? I can do the translations each time before merging.